reconcile/aws_ami_share.py

```diff
@@ -0,0 +1,86 @@
+import logging
+from typing import Any
+
+from reconcile import queries
+
+from reconcile.utils.aws_api import AWSApi
+
+QONTRACT_INTEGRATION = "aws-ami-share"
+MANAGED_TAG = {"Key": "managed_by_integration", "Value": QONTRACT_INTEGRATION}
+
+
+def filter_accounts(accounts: list[dict[str, Any]]):
```

Comment:
According to style guide, abstract collection types should be used in favor of concrete collection types for method/function arguments.

```
def filter_accounts(accounts: Iterable[Mapping[str, Any]]) ->  list[dict[str, Any]]:
```

Reply:
thanks!

Reply:
updated in https://github.com/app-sre/qontract-reconcile/pull/2216/commits/e406c89616e1fa6f666b74e75f3815c0524c1b08